### DISCUSSION TITLE
文档吐槽大会 [Documentation Discussion]

### DISCUSSION THREAD

**Discussion by zengyh1900** [open-mmlab/mmediting#1429] [General]
Let's talk about anything about the annoying documentation from MMEditing!
DO NOT let them waste your time!

master branch: https://mmediting.readthedocs.io/en/latest/
1.x branch: https://mmediting.readthedocs.io/en/1.x/
who can help: @zengyh1900 @Z-Fran @liuwenran @plyfager @LeoXing1996 @VongolaWu 

--------------
让我们尽情吐槽烦人的 MMEditing 文档吧！别再让他们浪费你的时间！

master 分支相关文档：https://mmediting.readthedocs.io/zh_CN/latest/index.html
1.x 分支相关文档：https://mmediting.readthedocs.io/en/1.x/
谁可以提供帮助：@zengyh1900 @Z-Fran @liuwenran @plyfager @LeoXing1996 @VongolaWu

**Comment by youqingxiaozhua** [comment]
请问 `data_preprocessor`是在哪里调用的呢？想用data_preprocessor来加padding，但是好像inference的时候并没有过data_preprocessor？

**Reply by jinxianwei** [reply]
https://github.com/open-mmlab/mmagic/pull/2054

**Reply by jinxianwei** [reply]
https://github.com/open-mmlab/mmagic/pull/2055
